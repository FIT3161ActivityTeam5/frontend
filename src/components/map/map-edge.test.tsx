import React from 'react';
import renderer from 'react-test-renderer';

import MapEdge from './map-edge';

describe('<MapEdge />', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
          <MapEdge
              x1={0}
              y1={0}
              x2={1}
              y2={1}
          />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
});
