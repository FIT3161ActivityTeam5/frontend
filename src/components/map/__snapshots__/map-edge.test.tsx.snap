// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapEdge /> renders correctly 1`] = `
<RNSVGLine
  propList={
    Array [
      "stroke",
      "strokeWidth",
    ]
  }
  stroke={4283178389}
  strokeWidth={2}
  x1={0}
  x2={1}
  y1={0}
  y2={1}
/>
`;
